src/Aspire.Dashboard/ResourceService/Partials.cs

```diff
@@ -107,8 +108,38 @@ ImmutableArray<VolumeViewModel> GetVolumes()
         ImmutableArray<CommandViewModel> GetCommands()
         {
             return Commands
-                .Select(c => new CommandViewModel(c.Name, MapState(c.State), c.DisplayName, c.DisplayDescription, c.ConfirmationMessage, c.Parameter, c.IsHighlighted, c.IconName, MapIconVariant(c.IconVariant)))
+                .Select(c =>
+                {
+                    var (displayName, displayDescription) = GetDisplayNameAndDescription(c.Name, c.DisplayName, c.DisplayDescription);
+                    return new CommandViewModel(c.Name, MapState(c.State), displayName, displayDescription, c.ConfirmationMessage, c.Parameter, c.IsHighlighted, c.IconName, MapIconVariant(c.IconVariant));
+                })
                 .ToImmutableArray();
+
+            // Use custom localizations for built-in lifecycle commands
+            static (string DisplayName, string DisplayDescription) GetDisplayNameAndDescription(string commandName, string displayName, string description)
```

Comment:
I thought we were going to add localized values to the proto, so that other resource services (like that in ACA) can also have localized command strings.

Reply:
I see James suggested this approach in the linked issue. I don't think it'd be too bad to add support for this to the proto. I would expect ACA to have loc requirements. cc @snehapar9

Reply:
This is a temporary solution for 9.1, we can improve this later @drewnoakes